conf/config-default.yaml

```diff
@@ -266,6 +266,7 @@ nginx_config:                     # config for render the template to generate n
       introspection: 10m
       access-tokens: 1m
       ext-plugin: 1m
+      tars: 1m
```

Comment:
Let's make it configurable in the next PR.

Reply:
> Need to handle this.

Sorry，I did not get your point
Your means we can keep using this db hanlder event get last_db_error ?

Reply:
We need to close the handler when getting last_db_error?

Reply:
You mean we need to decide whether to close or not based on the value of error ?

Currently the case of error=='again' has been handled
But So Sorry for I don't know what other error values do not need to be closed.

Reply:
According to the example, we need to close if we don't keepalive?
https://github.com/openresty/lua-resty-mysql#synopsis
```
				-- put it into the connection pool of size 100,
                -- with 10 seconds max idle timeout
                local ok, err = db:set_keepalive(10000, 100)
                if not ok then
                    ngx.say("failed to set keepalive: ", err)
                    return
                end

                -- or just close the connection right away:
                -- local ok, err = db:close()
                -- if not ok then
                --     ngx.say("failed to close: ", err)
                --     return
                -- end
```

Reply:
Sorry,  I  think this place doesn't actively call db:close() should be acceptable, 
[this link](https://forum.openresty.us/d/2151-f9a6aa2bb662b5ab77ee279e6dbd3ed6/4)  provides agentzh's view